src/olm/account/one_time_keys.rs

```diff
@@ -32,6 +32,14 @@ pub(super) struct OneTimeKeys {
     pub key_ids_by_key: HashMap<Curve25519PublicKey, KeyId>,
 }
 
+impl Drop for OneTimeKeys {
```

Comment:
Is this needed? All the values in the map will get dropped and zeroize is implemented for the type in the map.

Reply:
Yep, thanks for catching. What I actually should've done is remove the `Zeroize` impl for `OneTimeKeysPickle` to make everything symmetric. I'll make the change.